zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -0,0 +1,62 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/relaychain.tgz?raw=true"
```

Comment:
Could you please change the names of the files according to [this](https://github.com/paritytech/substrate/blob/master/zombienet/0001-basic-warp-sync/README.md):
> `file-SHA1SUM.tgz`, where SHA1SUM is a sha1sum of the file.

Reply:
Filenames renamed